website/settings/defaults.py

```diff
@@ -578,6 +579,10 @@ class CeleryConfig:
                 'task': 'scripts.generate_sitemap',
                 'schedule': crontab(minute=0, hour=5),  # Daily 12:00 a.m.
             },
+            'deactivate_requested_accounts': {
+                'task': 'scripts.periodic.deactivate_requested_accounts',
+                'schedule': crontab(minute=0, hour=5),  # Daily 12:00 a.m.
```

Comment:
Check with Matt over the best time to run this!

Reply:
Matt said this is too short to really worry about, so we good!